features/view/ascii_circuit_parser.feature.md

```
@@ -0,0 +1,278 @@
+# Feature: qni view の ASCII 回路パーサ
+
+qni-cli のユーザとして、Markdown feature から回路図を入力できるように、
+qni view の ASCII 回路パーサで circuit.json を作りたい。
+
+## Scenario: 空回路の ASCII 回路図を読み込める
+
+- When 次の回路図を読み込もうとする:
+
+  ```text
+  q0: ─────
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)